client/src/components/CollapsibleTable.jsx

```diff
@@ -109,6 +110,13 @@ function Row(props) {
             <DeleteIcon />
           </IconButton>
         </TableCell>
+        <TableCell style={{ width: '3%' }}> 
+        <Link to={`/teacher/copyProposal/${row.p.id}`}>
+          <IconButton aria-label="copy" onClick={() => { }} disabled={proposalAccepted  || row.p.status === 'assigned'}>
```

Comment:
What does this onClick function do?

Reply:
When the teacher click on IconButton he will be send to the route /teacher/copyProposal/:proposalId where there is the form of the proposal and the can add this

Reply:
For compatibility and coherence reasons, don't use Link to, and use Navigate.

You need to import it and use as follows:
````js
const navigate = useNavigate();
<IconButton ... onClick={ () => navigate('...');} ...>
```